src/Controller/DataController.java

```diff
@@ -427,9 +427,7 @@ static private HubFile processNewHubFile(NodeList nList) throws Exception
             ConsoleIO.setConsoleMessage("Attempting to import " + Integer.toString(assetList.size())
                     + " items to VersionControl...", true);
             ConsoleIO.setConsoleMessage("Starting with " + VersionControlEntity.libraryReport() + " entries");
-
```

Comment:
This should likely have remained as a blank line to maintain readability.

Reply:
This has been rectified in current commits, white space here is fixed and located at line 417 in DataController